libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/ActivityUtils.java

```diff
@@ -46,4 +47,8 @@ public static void showKeyboard(@Nullable final View view) {
                 (InputMethodManager) view.getContext().getSystemService(Context.INPUT_METHOD_SERVICE);
         inputMethodManager.showSoftInput(view, InputMethodManager.SHOW_IMPLICIT);
     }
+
+    public static boolean isDeepLinking(Intent intent) {
+        return Intent.ACTION_VIEW.equals(intent.getAction());
```

Comment:
This comment is quite confusing sorry:D... Basically what I wanted to propose is to manually add `intent.putExtraString("deeplinking", true);` (probably in `DeepLinkingIntentReceiverActivity`) instead of relying on ACTION_VIEW which isn't strictly tied to deeplinking.